install/README.md

```diff
@@ -167,6 +172,73 @@ minikube start --kubernetes-version v1.9.4 --vm-driver virtualbox \
+
+# Create the AKS cluster - this might take some time. Type 'az aks create -h' to see all available options
+# The following command will create a single Node AKS cluster. Node size is Standard A1 v1 and Kubernetes version is 1.9.6. Plus, SSH keys will be generated for you, use --ssh-key-value to provide your values
+az aks create --resource-group $AKS_RESOURCE_GROUP --name $AKS_NAME --node-count 1 --generate-ssh-keys --node-vm-size Standard_A1_v2 --kubernetes-version 1.9.6 --enable-rbac
+
+# Install kubectl
+sudo az aks install-cli
+
+# Get credentials for your new AKS cluster
+az aks get-credentials --resource-group $AKS_RESOURCE_GROUP --name $AKS_NAME
+```
+
+Alternatively, you can use the [Azure Portal](https://portal.azure.com) to create a new AKS cluster [(instructions)](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough-portal).
+
+### Allowing UDP traffic
+
+For Agones to work correctly, we need to allow UDP traffic to pass through to our AKS cluster. To achieve this, we must update the NSG (Network Security Group) with the proper rule. A simple way to do that is:
+
+* Login to the Azure Portal
+* Find the resource group where the AKS resources are kept, which should have a name like `MC_resourceGroupName_AKSName_westeurope`. Alternative, you can type `az resource show --namespace Microsoft.ContainerService --resource-type managedClusters -g $AKS_RESOURCE_GROUP -n $AKS_NAME -o json | jq .properties.nodeResourceGroup`
+* Find the Network Security Group object, which should have a name like `aks-agentpool-********-nsg`
+* Select **Inbound Security Rules**
+* Select **Add** to create a new Rule with **UDP** as the protocol and **7000-8000** as the Destination Port Ranges. Pick a proper name and leave everything else at their default values
+
+Alternatively, you can use the following command, after modifying the `RESOURCE_GROUP_WITH_AKS_RESOURCES` and `NSG_NAME` values:
+
+```bash
+az network nsg rule create \
+  --resource-group RESOURCE_GROUP_WITH_AKS_RESOURCES \
+  --nsg-name NSG_NAME \
+  --name AgonesUDP \
+  --access Allow \
+  --protocol Udp \
+  --direction Inbound \
+  --priority 520 \
+  --source-port-range "*" \
+  --destination-port-range 7000-8000
+  ```
+
+### Creating and assigning Public IPs to Nodes
+
+Nodes in AKS don't get a Public IP by default. To assign a Public IP to a Node, find the Resource Group where the AKS resources are installerd on the [portal](https://portal.azure.com) (it should have a name like `MC_resourceGroupName_AKSName_westeurope`). Then, you can follow the instructions [here](https://blogs.technet.microsoft.com/srinathv/2018/02/07/how-to-add-a-public-ip-address-to-azure-vm-for-vm-failed-over-using-asr/) to create a new Public IP and assign it to the Node/VM. For more information on Public IPs for VM NICs, see [this document](https://docs.microsoft.com/azure/virtual-network/virtual-network-network-interface-addresses).
+
+Continue to [Installing Agones](#installing-agones).
+
```

Comment:
For AKS, did you need to do anything for RBAC?

Reply:
Thank you! So, I can confirm that nothing else needs to be set for RBAC apart from the existing instructions/commands, just did a quick test by creating a cluster from scratch with RBAC enabled, installed Agones and quickstarting a [game server](https://github.com/GoogleCloudPlatform/agones/blob/master/docs/create_gameserver.md), having a successful ACK. Apparently, the Agones controllers are working just fine. 
Anything else I should check to make sure Agones RBAC stuff is working as intended?

Reply:
Hello @dgkanatsios ,
if you created the gameserver in the default namespace and the controller lived in the agones-system you have tested RBAC, if RBAC was active it wouldn't work. Assuming the pods of the gameserver was 2/2 active.

Reply:
also I think @markmandel was wondering if the kubectl config fetched by `az aks get-credentials` get a cluster-admin role by default, I guess the answer is yes again since you were able to install agones, is this correct ?

Reply:
Also, I vaguely remember some conversation about whether you needed the AKS equivalent of

```
kubectl create clusterrolebinding cluster-admin-binding \
  --clusterrole cluster-admin --user `gcloud config get-value account`
```
Did you end up needing to do that? If not, we should probably also make that clear.

Reply:
thanks for your comments @Kuqd and @markmandel. To answer your questions
- the gameserver is created in the default namespace
```bash
dgkanatsios@digkanat:~$ k get po --all-namespaces
NAMESPACE       NAME                                    READY     STATUS    RESTARTS   AGE
agones-system   agones-controller-cfccbcc6b-v9xr2       1/1       Running   0          15h
default         simple-udp-msmr9                        2/2       Running   0          15h
kube-system     azureproxy-899885bfb-5rsql              1/1       Running   2          15h
kube-system     heapster-56c6f9566f-pkbxw               2/2       Running   0          15h
kube-system     kube-dns-v20-7c556f89c5-75qzb           3/3       Running   0          15h
kube-system     kube-dns-v20-7c556f89c5-msfbc           3/3       Running   0          15h
kube-system     kube-proxy-rsjpn                        1/1       Running   0          15h
kube-system     kube-svc-redirect-gtghx                 1/1       Running   0          15h
kube-system     kubernetes-dashboard-5ffc5c5558-tdkwh   1/1       Running   4          15h
kube-system     tunnelfront-856cdb8bb7-khs86            1/1       Running   0          15h
```
- `az aks get-credentials` fetches cluster user credentials ([reference](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-get-credentials)), Agones CRDs and rest as well as test game server are installed with kubectl with these credentials
- no, a clusterrolebinding creation wasn't necessary, this is way I'm skipping the 'configure RBAC section'. Open to rewrite this in a better way, any suggestions?
Some commands output as well
```bash
dgkanatsios@digkanat:~$ kubectl get clusterrolebindings | grep -v system
NAME                                                   AGE
agones-controller-access                               15h
cluster-admin                                          15h
heapster                                               15h
dgkanatsios@digkanat:~$ kubectl get clusterroles | grep -v system
NAME                                                                   AGE
admin                                                                  15h
agones-controller                                                      15h
agones-sdk                                                             15h
cluster-admin                                                          15h
edit                                                                   15h
view                                                                   15h
dgkanatsios@digkanat:~$ kubectl describe clusterrole cluster-admin
Name:         cluster-admin
Labels:       kubernetes.io/bootstrapping=rbac-defaults
Annotations:  rbac.authorization.kubernetes.io/autoupdate=true
PolicyRule:
  Resources  Non-Resource URLs  Resource Names  Verbs
  ---------  -----------------  --------------  -----
  *.*        []                 []              [*]
             [*]                []              [*]
```
- some gameserver command output
```bash
dgkanatsios@digkanat:~$ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/master/examples/simple-udp/server/gameserver.yaml
gameserver.stable.agones.dev "simple-udp" created
dgkanatsios@digkanat:~$ k get gs
NAME         AGE
simple-udp   21s
dgkanatsios@digkanat:~$ k describe gs simple-udp
Name:         simple-udp
Namespace:    default
Labels:       <none>
Annotations:  kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"stable.agones.dev/v1alpha1","kind":"GameServer","metadata":{"annotations":{},"name":"simple-udp","name
space":"default"},"spec":{"contain...
API Version:  stable.agones.dev/v1alpha1
Kind:         GameServer
Metadata:
  Cluster Name:
  Creation Timestamp:  2018-06-18T07:07:17Z
  Finalizers:
    stable.agones.dev
  Generation:        0
  Resource Version:  64414
  Self Link:         /apis/stable.agones.dev/v1alpha1/namespaces/default/gameservers/simple-udp
  UID:               3c38fb5f-72c6-11e8-b304-beff59d82c7f
Spec:
  Container:       simple-udp
  Container Port:  7654
  Health:
    Failure Threshold:      3
    Initial Delay Seconds:  5
    Period Seconds:         5
  Host Port:                7849
  Port Policy:              dynamic
  Protocol:                 UDP
  Template:
    Metadata:
      Creation Timestamp:  <nil>
    Spec:
      Containers:
        Image:  gcr.io/agones-images/udp-server:0.1
        Name:   simple-udp
        Resources:
Status:
  Address:    137.117.171.57
  Node Name:  aks-nodepool1-26745675-0
  Port:       7849
  State:      Ready
Events:
  Type    Reason          Age   From                   Message
  ----    ------          ----  ----                   -------
  Normal  PortAllocation  2m    gameserver-controller  Port allocated
  Normal  Creating        2m    gameserver-controller  Pod simple-udp-qnmqx created
  Normal  Starting        2m    gameserver-controller  Synced
  Normal  Ready           1m    gameserver-controller  Address and Port populated
dgkanatsios@digkanat:~$ nc -u 137.117.171.57 7849
HALO
ACK: HALO
^C
```

thanks!